hat/assets/js/apps/Iaso/domains/users/components/Filters.js

```diff
@@ -111,6 +115,19 @@ const Filters = ({ baseUrl, params }) => {
                         />
                     </Box>
                 </Grid>
+                <Grid item xs={2}>
```

Comment:
Please use `xs={3}` like the other columns, otherwise  this looks a bit weird.
<img width="1644" alt="Screenshot 2022-12-01 at 15 34 44" src="https://user-images.githubusercontent.com/12494624/205080547-af3d7017-e321-451e-abe5-04fe0e01b1fb.png">

Reply:
Done